PublicFolders/ValidateEXOPFDumpster.ps1

```diff
@@ -364,7 +400,35 @@ function ExtractLog {
     if (!(Test-Path  "$ExportPath\logs_$ts")) {
         mkdir "$ExportPath\logs_$ts" -Force | Out-Null
     }
-    $PublicFolderInfo | Export-Clixml -Path "$ExportPath\logs_$ts\PublicFolderInfo.xml"
+    $PublicFolderInfo | Export-Clixml -Path "$ExportPath\logs_$ts\PublicFolderInfo$($PublicFolderInfo.PublicFolder.Name).xml"
+}
+function AskForFeedback {
+    Write-Host "Please rate the script experience & tell us what you liked or what we can do better over https://aka.ms/PFDumpsterFeedback!" -ForegroundColor Cyan
+    "Please rate the script experience & tell us what you liked or what we can do better over https://aka.ms/PFDumpsterFeedback!" | Out-File $ExportPath\$Script:ReportName -Append
```

Comment:
I would recommend avoiding writing out strings the same thing multiple times. Store it as a variable and then call that variable.

Reply:
Done.